gate-plugins/src/main/kotlin/com/netflix/spinnaker/gate/plugins/web/PluginService.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.plugins.web
+
+import com.netflix.spinnaker.gate.services.TaskService
+import com.netflix.spinnaker.security.AuthenticatedRequest
+import org.springframework.stereotype.Service
+
+@Service
+class PluginService(
```

Comment:
Added this so it can be used between `PluginInfoController` and `PluginPublishController`.